packages/form-state-manager/src/types/compose-validators.d.ts

```diff
@@ -1,6 +1,13 @@
 import { Validator } from './validate';
 import AnyObject from './any-object';
 
-export type ComposeValidators = (validators: Validator[]) => (value: any, allValues: AnyObject) => Promise<any> | any;
+export interface WarningObject<T = string | undefined> {
+  type: string;
```

Comment:
I think this should a list of currently supported states. Right now it's just a "warning"

Reply:
Yeah.. right now it's just one exact string :smile: